src/pyosmeta/cli/update_contributors.py

```diff
@@ -0,0 +1,77 @@
+import pickle
+
+from pyosmeta.contributors import ProcessContributors
+from pyosmeta.file_io import clean_export_yml, get_api_token, load_website_yml
+
+# TODO: will this still run in gh actions??
+# TODO: add update=True like i did for update_reviews
+# TODO: still need to add a flag to not update specific fields
+# TODO: if i use composition and there are helpers in a class
+# that are used in a method that i call via composition are the helpers
+# still available?
+
+
+def main():
+    GITHUB_TOKEN = get_api_token()
+    update_all = False
+
+    # TODO - maybe add these as an attr in the contribs class?
+    json_files = [
```

Comment:
maybe would could have a variable to store `https://raw.githubusercontent.com/pyOpenSci` so you don't need to repeat it everytime

Reply:
fixed!  i also made a small list comprehension!

```
base_url = "https://raw.githubusercontent.com/pyOpenSci/"
    end_url = "/main/.all-contributorsrc"
    repos = [
        "python-package-guide",
        "software-peer-review",
        "pyopensci.github.io",
        "software-review",
        "update-web-metadata",
    ]
    json_files = [base_url + repo + end_url for repo in repos]
```